mountpoint-s3/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Unreleased
+
+Breaking changes:
+
+* Logging to disk is now disabled by default.
+  Logs will not longer be written to `$HOME/.mountpoint-s3/` and should be configured using `--log-directory <DIRECTORY>`.
```

Comment:
I think what we probably actually want here is to start drafting v1.0.0, with a summary of breaking changes we've made during the pre-release period.

Reply:
That's effectively what I'm going for here with the "unreleased" heading, which will eventually be v1.0.0 if there's no minor version bump before it.

Was there a specific change you had in mind?

Reply:
No, just that we should go back and add the ones we've missed before release.